### DISCUSSION TITLE
Encounter exception when create collection:locale::facet::_S_create_c_locale name not valid

### DISCUSSION THREAD

**Discussion by TuJianwei** [milvus-io/milvus#4772] [Q&A and General discussion]
For help.
Used  milvus-0.10.3-cpu
And I start milvus server in docker. i am not root user so when i start /var/lib/milvus/scripts/start_server.sh that I found the error "can not create wal and db ".
So I add the comman "RUN mkdir /var/lib/milvus/db && mkdir /var/lib/milvus/wal && chown -R {user} /var/lib/milvus
" in dockerfile. And It worked, the milvus server is started.
Then I use milvus-java-sdk create a collection and create index. But the error logs is 
`[2021-03-03 19:01:36,060][ERROR][ENGINE][HandleException][reqsched_thread] Encounter exception when create collection:locale::facet::_S_create_c_locale name not valid
[2021-03-03 19:01:36,060][ERROR][SERVER][TakeToExecute][reqsched_thread] Request failed with code: Database error: Encounter exception when create collection:locale::facet::_S_create_c_locale name not valid`
and there is no tables in /var/lib/milvus/db.

The nomal file is 
![image](https://user-images.githubusercontent.com/20569448/109799035-782a4580-7c56-11eb-958c-1914543549ed.png)

**Comment by TuJianwei** [comment]
![image](https://user-images.githubusercontent.com/20569448/109799204-b293e280-7c56-11eb-97b0-2a5fb1357351.png)

**Comment by yhmo** [comment]
The reason is "the locale of your OS is NOT English". Please read the below link:
https://dev.to/nabbisen/microsoft-sql-server-client-troubleshooting-about-locale-facet-screateclocale-name-not-valid-5gmj

Try the commands in terminal:
```
export LANG=en_US.UTF-8
locale-gen en_US.UTF-8
```
And restart milvus server.

**Reply by TuJianwei** [reply]
Yeah，It worked. 
Really appreciate your geneous help.
Best wishes!
